HISTORY.txt

```diff
@@ -85,3 +85,5 @@
                          to enable compatibility with Mol* and other tools (which requires >= 0.3.0, in that format);
                          Update package version format to `major.minor.patch` (!!! MUST USE THIS FORMAT MOVING FORWARD !!!)
 17-Jan-2024    V0.85.1 - Set BinaryCIF version tag to specifically 0.3.0 to ensure compatibility with Mol* and other tools
+23-Feb-2024    V0.86   - BinaryCIF writer properly tracks integer types to ensure
```

Comment:
Also updating this version to be `major.minor.patch`

Reply:
See new commit